Personel/Shubhangi/problem solving/8sept/problem1.c

```diff
@@ -0,0 +1,19 @@
+#include <stdio.h>
+//function declaration
```

Comment:
write the above comment infront of declaration so that other can understand what that particular code function means